schemas/udmi/pointset.json

```diff
@@ -30,17 +30,23 @@
   "definitions": {
     "point_property_names": {
       "type": "object",
-      "propertyNames": {
-        "oneOf": [
-          {
-            "enum": [
-              "present_value"
-            ]
+      "additionalProperties": false,
+      "properties": {
+        "present_value": {
+          "type": ["number", "string", "boolean"]
+        },
+        "fittings_value": {
```

Comment:
Why using fittings_value as a name of this item? It looks like you're creating an array of items so why not just call it array or something similar and abstract it?

Reply:
I think as we delve deeper we will find more examples like this where we have complex objects that do not have a simple 'value'. Time Schedules are a collection of start/stop times in a time/date calendar; Lighting control includes 'scenes' which define light levels for different luminaires. In programming they would an object. Maybe that is a term worth considering?